openquake/server/views.py

```diff
@@ -198,6 +198,15 @@ def get_engine_version(request):
 
 @cross_domain_ajax
 @require_http_methods(['GET'])
+def get_engine_new_version(request):
+    """
+    Return a string with if new versions have been released
+    """
+    return HttpResponse(engine.check_obsolete_version(None))
+
```

Comment:
Please change `check_obsolete_version` and remove the argument `oqparam` which is not used anyway